docs/decisions/0001-record-architecture-decisions.rst

```diff
@@ -0,0 +1,32 @@
+1. Record Architecture Decisions
```

Comment:
Do we need an ADR that says we'll write ADRs?

Reply:
No.  I removed this.